examples/distributed/distributed_debate.py

```diff
@@ -84,37 +83,46 @@ def run_main_process(parsed_args: argparse.Namespace) -> None:
     agentscope.init(
         model_configs="configs/model_configs.json",
     )
-    pro_agent = RpcDialogAgent(
-        name="Pro",
-        host=parsed_args.pro_host,
-        port=parsed_args.pro_port,
-        launch_server=False,
-    )
-    con_agent = RpcDialogAgent(
-        name="Con",
-        host=parsed_args.con_host,
-        port=parsed_args.con_port,
-        launch_server=False,
-    )
-    judge_agent = RpcDialogAgent(
-        name="Judge",
-        host=parsed_args.judge_host,
-        port=parsed_args.judge_port,
-        launch_server=False,
-    )
+    with open(
```

Comment:
Can we read the agents.config in `agentscope.init` function? Or as the following:

Reply:
```
pro_agent, con_agent, judge_agent = agentscope.init(model_configs="xxx", agent_configs="xxx")

# Then to distributed
pro_agent = pro_agent.to_dist(host=xx, port=xx, launch_server=False)
...
```